### PR TITLE
Add badges component

### PR DESCRIPTION
**Pull request by aesha-spreadd**
Add 2 different types of badges

1) Image Badges
Screenshot 2023-12-26 at 10 25 17 AM

2) Cart Badges
Screenshot 2023-12-26 at 10 25 06 AM

- [x] Checking the code is duplicate or not?
- [x]  Is these code has bug
- [ ]  Is code is check by shubhum sir or ankit sir?
- [ ]  Are you going to merge this or not?

### CODE REVIEW

File: src/Badges/Badges.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { StyledBadgesComponent } from './Badges.Styled';
+
+const Badges = ({ type, variant, img, number, colorsEffect }) => {
```
Comment (tarakhant1297) [inline src/Badges/Badges.js]: @aeshapatel025 use same props name as antd

File: src/Badges/Badges.stories.js [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+};
+
+export const CartBadges = () => (
+  <div>
+    <StyledBadgesComponent>
+      <Badges
+        type="small"
+        variant="cart"
+        img="https://adorn-components.netlify.app/assets/images/cart.svg"
+        number={10}
```
Comment (tarakhant1297) [inline src/Badges/Badges.stories.js]: @aeshapatel025 same as above

File: src/Badges/Badges.stories.js [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+  </div>
+);
+
+export const ImageBadges = () => (
+  <div>
+    <StyledBadgesComponent>
+      <Badges
+        type="small"
+        variant="image"
+        colorsEffect="primary"
```
Comment (tarakhant1297) [inline src/Badges/Badges.stories.js]: @aeshapatel025 use color props for Customize Badge dot color

**Review by tarakhant1297 — CHANGES_REQUESTED** [review]
https://ant.design/components/badge follow props name as antd components

File: src/Badges/Badges.js (hunk 1 above)
Comment (aesha-spreadd) [reply src/Badges/Badges.js]: i have changes it

File: src/Badges/Badges.stories.js (hunk 3 above)
Comment (aesha-spreadd) [reply src/Badges/Badges.stories.js]: changes it

File: src/Badges/Badges.stories.js (hunk 2 above)
Comment (aesha-spreadd) [reply src/Badges/Badges.stories.js]: changes done!

File: src/Badges/Badges.stories.js [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+    </StyledBadgesComponent>
+  </div>
+);
+
+export const ImageBadges = () => (
+  <div>
+    <StyledBadgesComponent>
+      <Badges
+        size="small"
+        shape="image"
```
Comment (tarakhant1297) [inline src/Badges/Badges.stories.js]: @aeshapatel025 shape should be circle and square

**Review by tarakhant1297 — CHANGES_REQUESTED** [review]

**Comment by aesha-spreadd** [comment]
https://github.com/aeshapatel025/Strive_UI/pull/14

File: src/Badges/Badges.Styled.js [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+          span.badge-number {
+            background-color: ${theme?.[color]};
+            color: ${theme?.[color]};
+            top: ${color === 'primary' ? 17 : color === 'warning' ? 26 : 36}px;
+          }
+        `
+      : css`
+          span.badge-number {
+            top: 1px;
+            right: -3px;
```
Comment (ankitadrixus08) [inline src/Badges/Badges.Styled.js:48]: why in minus here @aeshapatel025

**Review by ankitadrixus08 — APPROVED** [review]

**Pull request merged**